src/controllers/Book.js

```diff
@@ -1,12 +1,54 @@
 const bcrypt = require('bcrypt-nodejs');
 const TokeController = require('../assets/Token');
+const Sequelize = require('sequelize');
 const Book = require('../models/Book');
+const Op = Sequelize.Op;
 
 module.exports = {
   async index(req, res) {
-    const books = await Book.findAll();
+    const id_user = req.userId;
+    try {
+      const book = await Book.findAll({
+        where: {
+          id_user: {
+            [Op.ne]: id_user
+          }
+        },
+        include: [
+          {association: 'books'}
+        ]
+      });
+      
+      if(!book)
+        return res.status(404).json({ error: 'Cannot find book' });
+      
+      return res.json(book);
+    }
+    catch (err) {
+      console.log(err);
+      return res.status(500).json({ error: 'Internal Server Error!'});
+    }
+  },
+
+  async indexById(req, res) {
+    const id_book = req.params.id;
 
-    return res.json(books);
+    try {
+      const book = await Book.findOne({
+        where: { 
+          id: id_book,
+        }
+      });
```

Comment:
Maybe we will need the association with the user, because when we see a single book on the app, we need some informations from the user.